sources/packages/web/src/types/contracts/aest/roles.ts

```diff
@@ -8,6 +8,7 @@ export enum Role {
   AESTManualTriggerReassessment = "aest-manual-trigger-reassessment",
   AESTReports = "aest-reports",
   AESTCreateInstitution = "aest-create-institution",
+  AESTEditCASSupplierInfo = "aest-edit-cas-supplier-info",
```

Comment:
Is this updated in the spreadsheet?
Please also ensure that it will be added to the release instructions.

Reply:
I will. I was just waiting for this to be approved.